microsoft/botframework-solutions
# [Virtual Assistant][TypeScript] Implement Virtual Assistant Tests

## Description
**Depends on #999** 

- Add **Virtual Assistant tests**
- Add **mock resources**
- Add **nock** as a dependency
- Add `rush test` script
- Add **nockFixtures**
- Update `.gitignore` to accept test bot files

Related to #769 

## Testing Steps

1. Go to `.\AI\solutions\Virtual-Assistant\src\typescript`
2. Run `rush update`
3. Run `rush rebuild`
4. Run `rush test` and check that **tests are passing correctly** (*)

(*) Actually, there is no tests in `bot-solution`, so before running `rush test` you need to leave empty the script of `test` in `package.json` of the `bot-solution` project.

##  Checklist
